doc/conf_ref.rst

```diff
@@ -59,6 +59,14 @@ configuration file by your distribution to override the DNF defaults.
  [main] Options
 ================
 
+.. _allow_vendor_change-label:
+
+``allow_vendor_change``
+    :ref:`boolean <boolean-label>`
+
+    If disabled dnf will stick to vendor when upgrading or downgrading rpms.
```

Comment:
If I understand it correctly, vendor stickiness doesn't work during downgrades.
I'm also not sure about distro-sync.
Could you double-check that and update the text accordingly?
Could you also add a warning that the option is only for the users that know what they're doing and that it doesn't apply to all DNF commands?